webcrypto/key.go

```diff
@@ -188,8 +188,10 @@ func newKeyGenerator(rt *sobek.Runtime, normalized Algorithm, params sobek.Value
 		kg, err = newHMACKeyGenParams(rt, normalized, params)
 	case ECDH, ECDSA:
 		kg, err = newECKeyGenParams(rt, normalized, params)
+	case RSASsaPkcs1v15, RSAPss, RSAOaep:
+		kg, err = newRsaHashedKeyGenParams(rt, normalized, params)
 	default:
-		return nil, errors.New("key generation not implemented for algorithm " + normalized.Name)
+		return nil, NewError(NotImplemented, "unsupported algorithm for key generation: "+normalized.Name)
```

Comment:
Nit, same approach as for the digest operation, we could try to make the error message even more explicit.

```suggestion
				validAlgorithms := []string{AESCbc, AESCtr, AESGcm, AESKw, HMAC, ECDH, ECDSA, RSASsaPkcs1v15, RSAPss, RSAOaep}
		return nil, NewError(
			NotImplemented,
			"unsupported key generation algorithm '"+normalized.Name+"', "+
				"accepted values are: "+strings.Join(validAlgorithms, ", "),
		)
```

Reply:
I agree that it's nice to make error messages actionable, but frankly, I'm not sure if the local list serves here the best. I'll try to iterate on that in the following PRs by combining the details from `isRegisteredAlgorithm`, which could serve as the single source of truth